crowbar_framework/config/initializers/delayed_job_config.rb

```diff
@@ -14,12 +14,27 @@
 # limitations under the License.
 #
 
+module Delayed
+  class Worker
+    if Rails.env.production?
+      @log_file = File.join(ENV["CROWBAR_LOG_DIR"], "background_jobs.log")
+    else
+      @log_file = File.join(Rails.root, "log", "background_jobs.log")
+    end
+    @@logger = Logger.new(@log_file)
```

Comment:
I believe this needs to be a class var as its shared around all classes and makes no sense to have copies around, plus I dont know other way of overriding it properly to use the same `log_file` without having to check for the `Rails.env.production`